plugins/plugins.gradle.kts

```diff
@@ -30,17 +23,11 @@ subprojects {
             create<MavenPublication>("plugin.${project.name}") {
                 groupId = "edu.wpi.first.shuffleboard.plugin"
                 artifactId = project.name
-                getWPILibVersion()?.let { version = it }
+                version = project.version as String
```

Comment:
Use `.toString()`.

Reply:
I don't think that's necessary since all project versions are going to be strings